src/app/pages/crypto-asset-list/crypto-asset-list.tsx

```diff
@@ -0,0 +1,21 @@
+import type { AllTransferableCryptoAssetBalances } from '@shared/models/crypto-asset-balance.model';
+
+import { CryptoAssetListItem } from './components/crypto-asset-list-item';
+import { CryptoAssetListLayout } from './components/crypto-asset-list.layout';
+
+interface CryptoAssetListProps {
+  cryptoAssetBalances: AllTransferableCryptoAssetBalances[];
+}
+export function CryptoAssetList({ cryptoAssetBalances }: CryptoAssetListProps) {
```

Comment:
I feel like it is a bit strange to have a `page` named as a list component? I feel like separating it out is a bit confusing to me just in that it is the first step to getting to the send form (much like the flows with Ledger). I am fine with the separation, but maybe the name should change?

Now that I'm thinking about it, isn't this essentially the same as the balances list except one filters for `canTransfer` and `isPressable` and navigates to the send form? Maybe the list could be a shared feature?

Reply:
Was thinking here, why do we even need a `Send` button when we have the balances list? Couldn't a user just click on an asset from that list and it could go to the send form?